src/character-sheet/CharacterSheet.js

```diff
@@ -0,0 +1,282 @@
+                    character.positionDice.diceType = 10;
+                    character.positionDice.total = character.level;
+                    character.positionDice.current = character.level;
+                    character.armorClass =
+                        character.armor.ac +
+                        character.abilities.dexMod *
+                            character.armor.addDexModToAC +
+                        character.shield.ac;
+                    break;
+                case "Fencer":
+                    character.abilities = FENCER_SCORES;
+                    character.bloodiedAbility = FENCER_BLOODIED_ABILITY;
+                    character.positionDice.diceType = 8;
+                    character.positionDice.total = character.level;
+                    character.positionDice.current = character.level;
+                    break;
+                case "Jack of All Trades":
+                    character.abilities = JACK_OF_ALL_TRADES_SCORES;
+                    character.bloodiedAbility =
+                        JACK_OF_ALL_TRADES_BLOODIED_ABILITY;
+                    character.positionDice.diceType = 8;
+                    character.positionDice.total = character.level;
+                    character.positionDice.current = character.level;
+                    break;
+                case "Caster":
+                    character.abilities = CASTER_SCORES;
+                    character.bloodiedAbility = CASTER_BLOODIED_ABILITY;
+                    character.positionDice.diceType = 6;
+                    character.positionDice.total = character.level;
+                    character.positionDice.current = character.level;
+                    break;
+                default:
+                    break;
+            }
+            character.position.base =
+                character.positionDice.diceType +
+                character.abilities.conMod +
+                character.level;
+            character.position.combatPool =
+                character.positionDice.diceType +
+                character.abilities.conMod +
+                character.level;
+            character.position.current =
+                character.positionDice.diceType +
+                character.abilities.conMod +
+                character.level;
+        } else if (name === "selectedClass") {
+            character[name] = value;
+            setProficiencySelections(DEFAULT_PROFICIENCY_SELECTIONS);
+            switch (value) {
+                case "Knight":
+                    character.proficiencies.strSavingThrow = true;
+                    character.proficiencies.dexSavingThrow = false;
+                    character.proficiencies.conSavingThrow = true;
+                    character.proficiencies.intSavingThrow = false;
+                    character.proficiencies.wisSavingThrow = false;
+                    character.proficiencies.chaSavingThrow = false;
+                    break;
```

Comment:
Could we have the other saving throws be defaulted to false and override the ones that will be used? So we dont have to specify each saving throw